src/core/load-fragment.c

```diff
@@ -4727,20 +4727,20 @@ int config_parse_exec_directories(
                         continue;
                 }
 
+                if ((!isempty(dest) || !isempty(flags)) && streq(lvalue, "ConfigurationDirectory")) {
```

Comment:
hmm, btw, why block "ro" for ConfigurationDirectory? Seems like a most reasonably thing to do.

Reply:
no reason, just that symlinks weren't enabled either

Reply:
maybe drop both restrictions, i see no reason why we'd not do symlinks for config dirs.